lib/gemstash/cli/start.rb

```diff
@@ -11,11 +11,17 @@ def initialize(cli)
 
       def run
         store_config
-        Puma::CLI.new(args).run
+        setup_logging
+        logger = Gemstash::Logging.raw_logger
+        Puma::CLI.new(args, Puma::Events.new(logger, logger)).run
       end
 
     private
 
+      def setup_logging
+        Gemstash::Logging.setup_logger(Gemstash::Env.log_file)
```

Comment:
I think this would be better off as
```
Gemstash::Logging.setup_logger(Gemstash::Env.base_file("server.log"))
```
and then drop `Gemstash::Env.log_file` method, along with the `:logfile` config key.

I'm worried about having too much configuration options for details that are unlikely to need to change... which grows in the amount we need to maintain down the road. Easier to strip as many config that really doesn't need to change and add them when someone has a strong use case for it (along these lines, I think I'd like to rip out some Puma config I originally put in).